projects/swimlane/ngx-ui/src/lib/components/slider/slider.component.ts

```diff
@@ -25,43 +26,14 @@ const edge = window.navigator.userAgent.indexOf('Edge') > -1;
 
 @Component({
```

Comment:
can we try to add `exportAs: 'ngxSlider'` when possible to our components, it allows for syntax like so which gives to component instance as a variable in the markup
```
<ngx-slider
  #slider="ngxSlider"
></ngx-slider>
```

Reply:
good call, added